docs/src/release_notes/v1.22.md

```diff
@@ -0,0 +1,82 @@
+# Release notes for CloudNativePG 1.22
+
+History of user-visible changes in the 1.22 minor release of CloudNativePG.
+
+For a complete list of changes, please refer to the
+[commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.22)
+on the release branch in GitHub.
+
+## Version 1.22.0
+
+**Release date:** Dec 21, 2023
+
+!!! Important "Important changes from previous versions"
+    This release introduces a significant change, disabling the default usage
+    of the `ALTER SYSTEM` command in PostgreSQL. For users upgrading from a
+    previous version who wish to retain the old behavior, please refer to the
+    [upgrade documentation](installation_upgrade.md) for detailed instructions.
+
+Features:
+
+- **Declarative Tablespaces**: Introducing the `tablespaces` stanza in the
+  `Cluster` spec, enabling comprehensive lifecycle management of PostgreSQL
+  tablespaces for enhanced vertical scalability (#3410).
+
+- **Temporary Tablespaces**: Adding the `.spec.tablespaces[*].name.temporary`
```

Comment:
should this be `.spec.tablespaces[*].temporary`, we replace the map to slice for tablespaces